lms/djangoapps/courseware/views/index.py

```diff
@@ -644,9 +644,9 @@ def show_courseware_mfe_link(user, staff_access, course_key):
     # course team preview CourseWaffleFlag for this course *or* if we've turned
     # on the redirect for your students.
     mfe_enabled_for_course_team = COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW.is_enabled(course_key)
-    mfe_experiment_enabled_for_course = REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_experiment_on(course_key)
+    mfe_enabled_for_course = REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key)
 
-    if staff_access and (mfe_enabled_for_course_team or mfe_experiment_enabled_for_course):
+    if staff_access and (mfe_enabled_for_course_team or mfe_enabled_for_course):
```

Comment:
Nit: It's probably not expensive enough of a call to matter, but we _could_ refactor this to avoid the second lookup, if unnecessary (since most users won't have staff access of any kind).

```python
if staff_access:
    if mfe_enabled_for_course_team:
        return True
    mfe_enabled_for_course = REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key)
    if mfe_enabled_for_course:
        return True
return False
```

Reply:
On second thought, we can actually avoid _both_ of those `is_enabled` calls in the majority of cases:


```python
if staff_access:
    mfe_enabled_for_course_team = COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW.is_enabled(course_key)
    if mfe_enabled_for_course_team:
        return True
    mfe_enabled_for_course = REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key)
    if mfe_enabled_for_course:
        return True
return False
```

Reply:
@stvstnfrd : good catch; we may as well save a couple SQL queries every request. I pushed a fixup commit with this change.